### DISCUSSION TITLE
Issue with using client certificate

### DISCUSSION THREAD

**Discussion by shelu16** [projectdiscovery/nuclei#3867] [Q&A] [answered]
```console
nuclei -u https://xxx.xxx.xxx.xxxx  -tags cisco -cc mil.pem

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.5

                projectdiscovery.io

[FTL] Program exiting: if a client certification option is provided, then all three must be provided
```

how to use , as per your intruction 
![image](https://github.com/projectdiscovery/nuclei/assets/30596083/0a8cc11a-9792-4f14-8ef2-12c0f989bbea)

i am using as per ur mention but still showing error

**Comment by forgedhallpass** [comment, marked as answer]
@shelu16 as the error message states, for client certificate based authentication you are required to provide all three (`-ca`, `-cc`, `-ck`) flags.

That being said, there was an issue with the feature (please refer to #3800), which has been fixed in #3851, but it's not yet released, so you would need to build it yourself.
